### PR TITLE
Amended

### PR DESCRIPTION
**Pull request by AbuNavsa**
_Add ticket reference to Pull Request title: e.g. 'FS-123: Add content', if there is no ticket prefix with BAU_

### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,51 +8,22 @@
 [![Swagger](https://img.shields.io/badge/-Swagger-%23Clojure?style=for-the-badge&logo=swagger&logoColor=white)](https://funding-service-design-account-store-dev.london.cloudapps.digital/#/default/core.account.post_account_by_email)
```
Comment (srh-sloan) [inline README.md:8]: This link won't work any more as it's point at CF

File: README.md [hunk 2]
```diff
@@ -8,51 +8,22 @@
...
 ![Gunicorn](https://img.shields.io/badge/gunicorn-%298729.svg?style=for-the-badge&logo=gunicorn&logoColor=white)
 
-Repo for the funding service design account store.
+This repository is designed to handle the generating and providing of JSON payloads for accounts. It takes an email address and creates/returns a corresponding payload of account json
```
Comment (srh-sloan) [inline README.md:11]: Second sentence doesn't really make sense as an overview of the repo?

File: README.md [hunk 3]
```diff
@@ -8,51 +8,22 @@
...
+[Developer setup guide](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-setup.md)
 
-Takes a email address and creates/returns a corresponding payload of account json.
 
-## Prerequisites
-- python ^= 3.10
+This service depends on:
+- A postgres database
+- funding-service-design-application-store
+- funding-service-design-assessment-store
```
Comment (srh-sloan) [inline README.md:19]: It doesn't depend on these services.... Just the database

File: README.md [hunk 4]
```diff
@@ -8,51 +8,22 @@
...
-requirements-dev.txt and requirements.txt are updated using [pip-tools pip-compile](https://github.com/jazzband/pip-tools)
-To update requirements please manually add the dependencies in the .in files (not the requirements.txt files)
-Then run (in the following order):
-
-    pip-compile requirements.in
-
-    pip-compile requirements-dev.in
+# Data
+## Local DB Setup
+General instructions for local db development are available here: [Local database development](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/readmes/python-repos-db-development.md)
```
Comment (srh-sloan) [inline README.md:27]: Most of this DB stuff below is covered in the shared readme, please reword and remove the stuff that's covered elsewhere.

The env config stuff is useful tho and should be moved to one of the shared readmes as it applies to all repos

File: README.md [hunk 5]
```diff
@@ -96,46 +67,23 @@ You can run this api using a docker container. To build a image run the followin
...
-docker ps -a
-CONTAINER ID   IMAGE                       COMMAND                  CREATED          STATUS                    PORTS                    NAMES
-42633142c619   paketo-demofsd-app          "/cnb/process/web"       8 seconds ago    Up 7 seconds              0.0.0.0:8080->8080/tcp   peaceful_knuth
-```
-
 # Pipelines
 
 Place brief descriptions of Pipelines here
 
 * Deploy to Gov PaaS - This is a simple pipeline to demonstrate capabilities.  Builds, tests and deploys a simple python application to the PaaS for evaluation in Dev and Test Only.
```
Comment (srh-sloan) [inline README.md:82]: No longer used

**Review by srh-sloan — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (AbuNavsa) [reply README.md:19]: have removed

File: README.md (hunk 5 above)
Comment (AbuNavsa) [reply README.md:82]: Removed

File: README.md (hunk 2 above)
Comment (AbuNavsa) [reply README.md:11]: have shortened to just first sentence overview

**Review by srh-sloan — APPROVED** [review]

**Pull request merged**